Author: Josselin Mouette <joss@debian.org>
Description: Define AUTOSTARTDIR to /usr/share/gnome/xdg.
 This is needed because the autostart .desktop file changes with every
 release, as it's autogenerated with translations, and the path is
 hardcoded in the binary. Seriously, WTF?
Forwarded: no.

Index: libcryptui-3.2.0/daemon/Makefile.am
===================================================================
--- libcryptui-3.2.0.orig/daemon/Makefile.am	2011-08-02 10:48:04.000000000 +0200
+++ libcryptui-3.2.0/daemon/Makefile.am	2011-11-19 00:17:07.523144252 +0100
@@ -13,7 +13,7 @@
 	-DLOCALEDIR=\""$(localedir)"\" \
 	-DPIXMAPSDIR=\""$(pixmapsdir)"\" \
 	-DLIBCRYPTUI_API_SUBJECT_TO_CHANGE \
-	-DAUTOSTARTDIR=\""$(sysconfdir)/xdg/autostart"\" \
+	-DAUTOSTARTDIR=\""$(datadir)/gnome/autostart"\" \
 	-DSEAHORSE_UIDIR=\""$(uidir)"\"
 
 bin_PROGRAMS = seahorse-daemon
